hydroflow/tests/compile-fail/surface_source_interval_badarg.rs

```diff
@@ -0,0 +1,10 @@
+use hydroflow::{hydroflow_syntax, var_args};
+
+#[tokio::main]
+async fn main() {
+    let mut df = hydroflow_syntax! {
+        // Should be a `Duration`.
+        source_interval(5) -> for_each(std::mem::drop);
```

Comment:
any reason for this over null();?

Reply:
Oh hmm, I guess I've just always used `for_each(std::mem::drop)` in these compile-fail tests instead of `null()`. Not any strong reason. Although `null()` can be a bit weird since it can act as both a source a sink.